### PR TITLE
fix: schema typing

### PR DESCRIPTION
**Pull request by aarnphm**
Some errors wrt schema type annotations

### CODE REVIEW

File: typings/schema.pyi [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-from typing import Protocol, Tuple, TypeVar, Union, overload
+from types import BuiltinFunctionType, FunctionType
+from typing import Any, Callable, Dict, List
+from typing import Literal as LiteralType
+from typing import Optional as OptionalType
+from typing import Tuple, Union, overload
+
+GenericType = Union["Schema", "And", "Or", "Use", "Optional", "Regex", "Literal", "Const"]
+AcceptedDictType = Dict[Union[str, GenericType], Any]
+_CallableLike = Union[FunctionType, BuiltinFunctionType, Callable[..., Any]]
```
Comment (bojiang) [inline typings/schema.pyi:12]: `Callable` should already include the above. Any exceptions that requires this

Comment (bojiang) [reply typings/schema.pyi:12]: @aarnphm

Comment (aarnphm) [reply typings/schema.pyi:12]: Hmm I don't think so

Comment (bojiang) [reply typings/schema.pyi:12]: Any exception?

**Pull request merged**